osbs/build/build_requestv2.py

```diff
@@ -571,11 +572,16 @@ def _set_deadline(self):
 
         self.set_deadline(deadline_hours)
 
-    def adjust_for_repo_info(self):
+    def adjust_for_repo_info(self, release_env_var, release):
         if not self._repo_info:
             logger.warning('repo info not set')
             return
 
+        if release_env_var.value:
+            release_var = "ENV {}={}".format(release_env_var.value, release.value)
```

Comment:
Could the release value be None at this point? If so, should we warn about setting the env var with an empty value?